docs/asciidoc/static/deploying.asciidoc

```diff
@@ -0,0 +1,145 @@
+[[deploying-and-scaling]]
+== Deploying and Scaling Logstash
+
+The requirements for your Logstash installation change according to the scale demanded by your use case. In this 
+section, we will examine a range of Logstash installations in increasing order of complexity, starting from a minimal 
+viable installation and adding elements to the system as we go. The example deployments in this section write to an 
+Elasticsearch cluster, but Logstash can write to a large variety of {logstash}output-plugins.html[endpoints].
+
+// [float]
+[[deploying-minimal-install]]
+=== Getting Started: Minimal Installation
+
+The minimal Logstash installation has one Logstash instance and one Elasticsearch instance. These instances are 
+directly connected, with Logstash using an {logstash}input-plugins.html[_input plugin_] to ingest data and an 
+Elasticsearch {logstash}output-plugins.html[_output plugin_] to index the data in Elasticsearch, following the Logstash 
+{logstash}pipeline.html[_processing pipeline_]. A given Logstash instance has a fixed pipeline constructed at startup, 
+based on the instance’s configuration file. You must specify an input plugin. Output defaults to `stdout`, and the 
+filtering section of the pipeline, which is discussed in the next section, is optional.
+
+image::images/deploy_1.png[]
+
+// [float]
+[[deploying-filter-threads]]
+=== Adding a Filter and Threads
+
+Log data is typically unstructured, often contains extraneous information that isn’t relevant to your use case, and 
+sometimes is missing relevant information that can be derived from the log contents. A 
+{logstash}filter-plugins.html[filter plugin] can handle these situations by parsing the log into fields, removing 
```

Comment:
"You can use a filter plugin to parse the log data into fields, remove unnecessary information, and derive additional information from the existing fields."?